build/crds/neptune/federatedlearningjob_v1alpha1.yaml

```diff
@@ -0,0 +1,179 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: federatedlearningjobs.neptune.io
+spec:
+  group: neptune.io
+  names:
+    kind: FederatedLearningJob
+    plural: federatedlearningjobs
+    shortNames:
+      - fl
```

Comment:
flj is enough

Reply:
i think it can be remained whitch seems more intuitive

Reply:
then use `fl`, one short name is enough